diff --git a/Main.qml b/Main.qml
index 1234567..abcdefg 100644
--- a/Main.qml
+++ b/Main.qml
@@ -76,7 +76,7 @@ Pane {
             anchors.centerIn: form
             color: root.palette.window
             visible: config.HaveFormBackground == "true" ? true : false
-            opacity: config.PartialBlur == "true" ? 0.3 : 1
+            opacity: config.PartialBlur == "true" ? 0.15 : 0.2
             z: 1
         }
 